README.md

```diff
@@ -1,77 +1,246 @@
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command: just need a web brwosser 
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command: you just need a simple web browser to run this project for a test
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/henask12)
+- Twitter: [@twitterhandle](https://twitter.com/henok_mekonnen3)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/henok-mekonnen-491685188)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **Display Hello Microverse**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- CONTRIBUTING -->
-Contributing
-Contributions are what make the open source community such an amazing place to learn, inspire, and create. Any contributions you make are greatly appreciated.
 
-Fork the Project
-Create your Feature Branch (git checkout -b feature/hello-microverse)
-Commit your Changes (git commit -m 'Add some Hello microverse')
+## 🤝 Contributing <a name="contributing"></a>
 
-Push to the Branch (git push origin feature/hello-microverse)
-Open a Pull Request
-<!-- LICENSE -->
-License
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
 
-<!-- CONTACT -->
-Contact
-Hena - @henask12
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-Project Link: https://github.com/henask12/hello-microverse
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
-Acknowledgements
-Microverse for the amazing curriculum and community.
 
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank my Coding partner and My Code Reviewer
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
 
+This project is [MIT](./LICENSE) licensed.
 
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
![image](https://user-images.githubusercontent.com/78251927/234655928-68d55b18-b0d7-49e9-a473-900987caa557.png)

- [ ] The LICENSE link is broken. kindly consider adding a license to your repository. 

- [ ] Public repositories on GitHub are often used to share open-source software. For your repository to be open source, you'll need to license it so that others are free to use, change, and distribute the software.

- [ ] You can check here on [Adding a license to a repository](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) 🤗🛠

Reply:
added MIT License